tests/shipping/models/Delivery - No zipcode.model.js

```diff
@@ -0,0 +1,71 @@
+import { setup, visitAndClearCookies } from '../../../utils'
+import {
+  fillEmail,
+  getRandomEmail,
+  fillProfile,
+} from '../../../utils/profile-actions'
+import { goToPayment, selectCountry } from '../../../utils/shipping-actions'
+import {
+  completePurchase,
+  fillBillingAddress,
+  payWithCreditCard,
+} from '../../../utils/payment-actions'
+import { SKUS } from '../../../utils/constants'
+
+export default function test(account) {
+  describe(`Delivery - Credit Card - ${account}`, () => {
+    before(() => {
+      visitAndClearCookies(account)
+    })
+
+    it('with only delivery', () => {
+      const email = getRandomEmail()
+
+      setup({ skus: [SKUS.DELIVERY_ARG], account })
+
+      fillEmail(email)
+      fillProfile()
+
+      selectCountry('ARG')
+
+      cy.waitAndGet('#ship-addressQuery', 3000).type(
+        'Corrientes 240, Las Varillas, Córdoba, Argentina'
+      )
+
+      cy.get('.pac-item')
+        .first()
+        .trigger('mouseover')
+
+      cy.get('.pac-item')
+        .first()
+        .click()
```

Comment:
what do you think of intercepting the gmaps autocomplete response here to make sure it won't return a postal code?